sections/main-article.liquid

```diff
@@ -2,319 +2,289 @@
 
 <article class="article-template" itemscope itemtype="http://schema.org/BlogPosting">
   {%- for block in section.blocks -%}
-    {%- case block.type -%}
-      {%- when '@app' -%}
-        <div class="page-width page-width--narrow">
-          {% render block %}
-        </div>
-      {%- when 'featured_image'-%}
-        {%- if article.image -%}
-          <div class="article-template__hero-container" {{ block.shopify_attributes }}>
-            <div class="article-template__hero-{{ block.settings.image_height }} media"
-              itemprop="image"
-              {% if block.settings.image_height == 'adapt' and article.image %} style="padding-bottom: {{ 1 | divided_by: article.image.aspect_ratio | times: 100 }}%;"{% endif %}
-            >
-              <img srcset="{% if article.image.width >= 350 %}{{ article.image | img_url: '350x' }} 350w,{% endif %}
+  {%- case block.type -%}
+  {%- when '@app' -%}
+  <div class="page-width page-width--narrow">
+    {% render block %}
+  </div>
+  {%- when 'featured_image'-%}
+  {%- if article.image -%}
+  <div class="article-template__hero-container" {{ block.shopify_attributes }}>
+    <div class="article-template__hero-{{ block.settings.image_height }} media" itemprop="image" {% if
+      block.settings.image_height=='adapt' and article.image %}
+      style="padding-bottom: {{ 1 | divided_by: article.image.aspect_ratio | times: 100 }}%;" {% endif %}>
+      <img srcset="{% if article.image.width >= 350 %}{{ article.image | img_url: '350x' }} 350w,{% endif %}
```

Comment:
Here we would want to keep the original indentation for readability. It might be a setting in your code editor which will format the code when saving the file. You can probably disable it before saving as the only change we should see in this file is the reference to `month_year` changing to `date`.